### PR TITLE
Feature email sing in sing up

### PR DESCRIPTION
**Pull request by avoevodin**
#27
#28

### CODE REVIEW

File: brigadier/accounts/admin.py [hunk 1]
```diff
@@ -1,3 +1,80 @@
...
+    """A form for updating users. Includes all the fields on
+    the user, but replaces the password field with admin's
+    disabled password hash display field.
+
+    """
+    password = ReadOnlyPasswordHashField()
+
+    class Meta:
+        model = MyUser
+        fields = ('email', 'first_name', 'last_name', 'is_active', 'is_admin')
```
Comment (iliadmitriev) [inline brigadier/accounts/admin.py:48]: move to forms.py

File: brigadier/accounts/migrations/0002_create_public_group.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def create_public_group(apps, schema_editor):
 
     Group = apps.get_model('auth', 'Group')
     Permission = apps.get_model('auth', 'Permission')
-    User = apps.get_model('auth', 'User')
+    User = apps.get_model('accounts', 'MyUser')
```
Comment (iliadmitriev) [inline brigadier/accounts/migrations/0002_create_public_group.py:18]: remember you should never change migrations
when you want your db model to be change - create a new migration!

File: brigadier/brigadier/settings.py [hunk 3]
```diff
@@ -162,4 +162,15 @@
         '127.0.0.1'
     ]
 
+EMAIL_HOST = 'localhost'
+EMAIL_PORT = 1025
```
Comment (iliadmitriev) [inline brigadier/brigadier/settings.py:166]: all services connections should be configured via environment variables

**Review by iliadmitriev — CHANGES_REQUESTED** [review]
1) move forms from admin.py to forms.py
2) README.md: how to create a smtp server instance and configure its connection?
3) settings.py: all services connections should be configured via environment variables,
and email service is not an exception

File: brigadier/accounts/migrations/0002_create_public_group.py (hunk 2 above)
Comment (avoevodin) [reply brigadier/accounts/migrations/0002_create_public_group.py:18]: As it already have done, I won't correct this mistake. But I get this rule for the future.

**Comment by avoevodin** [comment]
Iliuxa, i need the next step 😂
image

**Comment by iliadmitriev** [comment]
You know the drill.
And cute pictures don't work.
Fix all the remarks, assign pull request and request a code review again.

**Comment by avoevodin** [comment]
> You know the drill.
> And cute pictures don't work.
> Fix all the remarks, assign pull request and request a code review again.

Do you mean the remark of migrations? Because this is the one i didn't fix

**Review by iliadmitriev — APPROVED** [review]

**Pull request merged**